apix-impl/src/integration-test/java/eu/xenit/apix/tests/search/SearchServiceTest.java

```diff
@@ -322,4 +321,31 @@ private void withLocale(Locale locale, AuthenticationUtil.RunAsWork<Object> work
      */
     abstract protected void withTestFacets(AuthenticationUtil.RunAsWork<Object> work) throws Exception;
 
+
+    @Test
+    public void testNotFilteredOut() {
+        SearchSyntaxNode node =
+                new QueryBuilder()
+                        .startOr()
+                        .property("{http://www.alfresco.org/model/system/1.0}node-dbid", "9223372036854775807")
```

Comment:
Can you use Long.MAX_VALUE instead of "9223372036854775807" so people know what the value means?